crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -95,9 +136,25 @@ impl From<RpoFalcon512ProcedureAcl> for AccountComponent {
         // Slot 0: Public key
         storage_slots.push(StorageSlot::Value(falcon.public_key.into()));
 
-        // Slot 1: Number of tracked procedures
+        // Slot 1: [num_tracked_procs, allow_unauthorized_output_notes,
+        // allow_unauthorized_input_notes, 0]
         let num_procs = falcon.auth_trigger_procedures.len() as u32;
-        storage_slots.push(StorageSlot::Value(Word::from([num_procs, 0, 0, 0])));
+        let allow_unauthorized_output_notes = if falcon.allow_unauthorized_output_notes {
+            1u32
+        } else {
+            0u32
+        };
+        let allow_unauthorized_input_notes = if falcon.allow_unauthorized_input_notes {
+            1u32
+        } else {
+            0u32
+        };
+        storage_slots.push(StorageSlot::Value(Word::from([
+            num_procs,
+            allow_unauthorized_output_notes,
+            allow_unauthorized_input_notes,
+            0,
+        ])));
```

Comment:
Nit: There is a conversion from [`bool` to `u32`](https://doc.rust-lang.org/std/primitive.bool.html#impl-From%3Cbool%3E-for-u32), so this could be written a bit more concisely (using `From` or `as`):

```rust
storage_slots.push(StorageSlot::Value(Word::from([
    num_procs,
    u32::from(falcon.allow_unauthorized_output_notes),
    u32::from(falcon.allow_unauthorized_input_notes),
    0,
])));
```